### PR TITLE
vim-patch:30e9b3c42567

### PR DESCRIPTION
**Pull request by VVKot**
Update runtime files
https://github.com/vim/vim/commit/30e9b3c4256710781c3bd64efb33f138e4e074b3

### CODE REVIEW

File: runtime/doc/starting.txt [hunk 1]
```diff
@@ -106,8 +106,8 @@ argument.
 		This can be used to find out where time is spent while loading
 		your |config|, plugins and opening the first file.
 		When {fname} already exists new messages are appended.
-		(Only available when compiled with the |+startuptime|
-		feature).
+		{only available when compiled with the |+startuptime|
```
Comment (VVKot) [inline runtime/doc/starting.txt:109]: Should I just remove this instead? It seems that other mentions of compile-time features are removed from docs.

**Comment by VVKot** [comment]
@janlazo are you the best person to review this?

**Comment by VVKot** [comment]
For history - this was previous attempted in #12214.

File: runtime/doc/starting.txt (hunk 1 above)
Comment (janlazo) [reply runtime/doc/starting.txt:109]: Sure.

**Comment by janlazo** [comment]
This is not a full port so it shouldn't use `vim-patch:` token.

**Comment by VVKot** [comment]
@janlazo I thought I ported all relevant changes; but seems there is one that we'll need previous patches. Is it fine to merge those (I removed vim-patch from commit message)? I'll put a separate PR for porting everything relevant to `textprop.txt`.

1. runtime/doc/cmdline.txt - noop for Neovim, none of these lines are present
2. runtime/doc/editing.txt - ported
3. runtime/doc/index.txt - this was already ported
4. runtime/doc/popup.txt - this file is not present in the Neovim, and it only updated date anyway
5. runtime/doc/sign.txt - noop, this line is not present in Neovim
5. runtime/doc/starting.txt - noop as it only changed lines that mention conditionally compiled features
6. runtime/doc/syntax.txt - ported
7. runtime/doc/tags - not relevant for Neovim
8. runtime/doc/textprop.txt - RELEVANT, needs https://github.com/vim/vim/commit/98aefe7c3250bb5d4153b994f878594d1745424e to be ported first
9. runtime/doc/todo.txt - not relevant for Neovim
10. runtime/doc/various.txt - noop change for Neovim
11. runtime/indent/tcl.vim - ported
12. runtime/menu.vim - ported

**Comment by VVKot** [comment]
@janlazo is this good to go? See my comment above.

**Comment by janlazo** [comment]
@VVKot You're right. Use the `vim-patch:` token. Redo the commit via `./scripts/vim-patch.sh` while keeping the existing changes and then I'll merge.

**Pull request merged**